content/rs/references/cli-utilities/crdb-cli/crdb/create.md

```diff
@@ -0,0 +1,114 @@
+---
+Title: crdb-cli crdb create
+linkTitle: create
+description: Creates an Active-Active database.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases:
+---
+
+Creates an Active-Active database.
+
+```sh
+crdb-cli crdb create --name <name>
+         --memory-size <maximum_memory>
+         --instance fqdn=<cluster_fqdn>,username=<username>,password=<password>[,url=<url>,replication_endpoint=<endpoint>]
+         --instance fqdn=<cluster_fqdn>,username=<username>,password=<password>[,url=<url>,replication_endpoint=<endpoint>]
+         [--port <port_number>]
+         [--no-wait]
+         [--default-db-config <configuration>]
+         [--default-db-config-file <filename>]
+         [--compression <0-6>]
+         [--causal-consistency { true | false } ]
+         [--password <password>]
+         [--replication { true | false } ]
+         [--encryption { true | false } ]
+         [--sharding { false | true } ]
+         [--shards-count <number_of_shards>]
+         [--shard-key-regex <regex_rule>]
+         [--oss-sharding { true | false } ]
+         [--oss-cluster { true | false } ]
+         [--bigstore { true | false }]
+         [--bigstore-ram-size <maximum_memory>]
+         [--with-module name=<module_name>,version=<module_version>,args=<module_args>]
+```
+
+### Prerequisites
+
+Before you create an Active-Active database, you must have:
+
+- At least two participating clusters
```

Comment:
That's actually not required, you can have an AA db on the same cluster

Reply:
Is that something we should change here? For example, use "should" instead of "must"? Since the purpose of Active-Active is to have multiple Active instances of the database, I'm not 100% sure we should change it.